src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -239,10 +238,8 @@ export const isWithinServicePeriod = (errors, fieldData, formData) => {
 export const validateDisabilityName = (err, fieldData) => {
   if (
     !LOWERED_DISABILITY_DESCRIPTIONS.includes(fieldData.toLowerCase()) &&
-    !EVSS_DISABILITY_NAME_REGEX.test(fieldData)
+    fieldData.length > 255
```

Comment:
Could you leave a comment here explaining why we're using a validator instead of `maxLength`?

Reply:
Yeah that's probably a good idea